Add eventbrite

theetrain opened the pull request:
Hello, thanks for maintaining this project!

* I've added a listing for eventbrite

![image](https://user-images.githubusercontent.com/12798751/65378540-c11b5380-dc87-11e9-8b3d-ba351bf46635.png)

theetrain commented on _data/social.yml:
```diff
@@ -34,7 +34,7 @@ websites:
       tfa: Yes
       sms: Yes
       doc: https://www.biggerpockets.com/forums/25/topics/232201
-     
```
Prettier must have detected these spaces and deleted them.

kmpoppe replied:
Hey @eetrain, would you be so kind as to revert that single line change, so that your PR only includes the Eventbrite change and the accompanying newline?
We will clear out superfluous characters at a later date.
Thanks!
// Kai

theetrain replied:
@kmpoppe 
No problem. As requested, this PR now only includes the Eventbrite change.

theetrain committed:
feat(social): add eventbrite listing

kmpoppe committed:
Update eventbrite.png

Compressed filesize using tinypng.com

Carlgo11 commented on _data/social.yml:
```diff
@@ -64,6 +64,13 @@ websites:
       img: ello.png
       tfa: No
 
+    - name: eventbrite
```
```suggestion
    - name: Eventbrite
```

kmpoppe replied:
Resolved with bd9aa67

Carlgo11 commented on _data/social.yml:
```diff
@@ -64,6 +64,13 @@ websites:
 
+    - name: eventbrite
+      url: https://www.eventbrite.com/
+      twitter: EventbriteHelp
```
```suggestion
      twitter: eventbritehelp
```

kmpoppe replied:
Resolved with bd9aa67

kmpoppe committed:
Update Eventbrite casing

Update casing
eventbrite => Eventbrite for the name
(Eventbrite Inc. due to https://www.eventbrite.com/support/articles/en_US/Troubleshooting/eventbrite-privacy-policy?lg=en_US)
as per
https://github.com/2factorauth/twofactorauth/pull/4033#discussion_r327802035

EventbriteHelp => eventbritehelp for twitter handle
as per
https://github.com/2factorauth/twofactorauth/pull/4033#discussion_r327802190
